workflow.md

```
@@ -33,22 +35,28 @@ Inside of normal we have a handful of tools, most are custom, and pluggable into
+| File ops      | File ops           | File ops      | File ops      | File ops              | Read only             |
+| Search        | Search             | Search        | Search        | Search                | Search (cli_*)        |
+| Tasks         | Tasks              | Tasks         | Tasks         | Tasks                 |                       |
+| Just runner   | Just runner        | Just runner   | Just runner   | Just runner           | Just runner (limited) |
+| Thoughts      | Thoughts           | Thoughts      | Thoughts      | Thoughts              | Thoughts (write only) |
+| GitHub PRs    | GitHub PRs         | GitHub PRs    | GitHub PRs    | GitHub PRs            |                       |
+| Sub-agents    | Sub-agents         | Sub-agents    | Sub-agents    | Sub-agents            | Reasoning model       |
+|               | GPT-5.4-oriented commands | + mcp_bash (shell) | + 9 Linear tools | + 22 Browser automation | + review_* tools |
 
 ### Commands Using Each
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)